assets/css/main.scss

```diff
@@ -147,3 +150,69 @@ div.xoverflow {
 }
 
 ol li ol { list-style-type: lower-latin; }
+.feature-yes strong {
+  border-bottom: 2px solid $compatibility-feature-yes;
+}
+.feature-no strong {
+  border-bottom: 2px solid $compatibility-feature-no;
+}
+.feature-neutral strong {
+  border-bottom: 2px solid lightgray;
+  color: gray;
+}
+
+.third-party-logo
+{
+  max-height:150px;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  padding:20px
+}
+
+:target {
+  background-color: #daf8fa;
+}
```

Comment:
I noticed this before and it's really cool, but it affects every target link on the site, which is maybe a bit broad.  E.g., on a preview of this branch, go to any newsletter page and add `#news` to the URL.  It'll look like:

![2019-07-29-09_36_36_118913358](https://user-images.githubusercontent.com/61096/62076858-6c59e000-b1e4-11e9-97a6-dac5eee28771.png)

That isn't bad IMO, but I'm unable to test all the different anchors on this site to ensure that they look good (however, I did test a bunch and none of them looked bad to me).  However, it might be worth limiting this to case where we expect it to look particularly good, such as with list items.  (I have no strong opinion.)

Reply:
Let's leave this in for now. If we notice issues with it, we can tighten the style to just list items.